master/internal/core.go

```diff
@@ -334,6 +345,7 @@ func (m *Master) Run() error {
 	// relative links in web pages hosted under these routes.
 	staticWebDirectoryPaths := map[string]bool{
 		"/docs":          true,
+		"/det":           true,
```

Comment:
**question:** can/should `/det` come from a config?

Reply:
hmm at the very least a const var sounds safer.

Reply:
that would consolidate it in master. but having a way to set it in one place for all the modules.. I'm not sure